doc/example-config

```diff
@@ -1018,15 +1007,25 @@
 #
 # prepopulate-query-feeds yes
 
-####  ssl-verify
+####  ssl-verifyhost
+#
+# If no, skip SSL certificate host's name validation.
+#
+# Syntax: [yes/no]
+#
+# Default value: yes
+#
+# ssl-verifyhost no
+
+####  ssl-verifypeer
 #
-# If no, skip SSL certificate validation.
+# If no, skip self-signed SSL certificate validation.
```

Comment:
This check isn't specific to self-signed certificates. It can be used to avoid errors with any untrusted CAs. Please undo this edit (and don't forget to update configcommands.dsv).